src/tsickle.ts

```diff
@@ -634,6 +634,11 @@ class Annotator extends ClosureRewriter {
    *     emit it as is and visit its children.
    */
   maybeProcess(node: ts.Node): boolean {
+    if (this.file.statements.indexOf(node as ts.Statement) !== -1) {
```

Comment:
Can you explain *why* we're doing this in the commit message? See go/cl-descriptions.

In particular, what does this fix? How does it relate to the idea of emitting mappings for each emitted string?

Conceptually, shouldn't we rather make sure we emit a mapping for each line of code that can actually be executed, assuming we have proper coverage for all tokens we emit? This seems like it'll still have issues for e.g. multiple statements in one line, or code nested deeper than the top level.